addons/base_automation/views/base_automation_views.xml

```diff
@@ -294,7 +294,7 @@
                     <field name="name"/>
                     <field name="model_id"/>
                     <separator/>
-                    <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
+                    <filter string="Include Archived" name="include_inactive" domain="[('active', 'in', [False, True])]"/>
```

Comment:
Thanks for the quick followup, but why the `name` change here? Can't do that in stable